test/lint/commit-script-check.sh

```diff
@@ -24,6 +24,10 @@ for commit in $(git rev-list --reverse $1); do
     if git rev-list -n 1 --pretty="%s" $commit | grep -q "^scripted-diff:"; then
         git checkout --quiet $commit^ || exit
         SCRIPT="$(git rev-list --format=%b -n1 $commit | sed '/^-BEGIN VERIFY SCRIPT-$/,/^-END VERIFY SCRIPT-$/{//!b};d')"
+        if echo "$SCRIPT" | grep --quiet "sed"; then
+            echo "Warning: do not use BSD sed syntax"
+            echo "Example: sed -i '' 's/foo/bar/' filename (Bad)"
```

Comment:
Would be better to only print this warning if the check fails, no?

Reply:
Which check? The script evaluation?

Reply:
With "check" I mean "script evaluation"